src/client/src/pricingOperations.ts

```diff
@@ -39,9 +39,13 @@ export const pricingServiceEpic = (pricingService$) => {
       })
   }
 
-  function getPrices(action$) {
+  function getPrices(action$, store) {
     return pricingStream$(action$)
       .mergeMap((symbols: Array<string>) => getCurrencyPairs(symbols, pricingService$))
+      .do( price => {
+        const update = {...price, ratePrecision: referenceDataService$.getCurrencyPair(price.symbol).ratePrecision }
```

Comment:
can you remove the $ suffix, as it looks confusing given its not a stream...

Reply:
Its not. $ is for observables. Not correct here. Will fix